.github/workflows/CI.yml

```
@@ -101,3 +103,30 @@ jobs:
           ./doc/chap1_mj.html
           ./doc/chap2_mj.html
           EOF
+
+      - name: "Test check_symlinks.sh"
+        shell: bash
+        working-directory: scripts
+        run: |
+          mkdir -p symlinks-dir
+          cd symlinks-dir
+          echo "THIS FILE IS NOT A SYMLINK" > actual.file
+          ../check_symlinks.sh > $RUNNER_TEMP/symlinks_run.txt # should not give error
+          ln -s actual.file sym.link
+          ../check_symlinks.sh > $RUNNER_TEMP/symlinks_run.txt || true # suppress expected error here
+          diff <(sort $RUNNER_TEMP/symlinks_run.txt) - <<EOF
+          ./sym.link
+          EOF
+
+      - name: "Test check_date.sh"
+        shell: bash
+        working-directory: scripts
+        run: |
+          if   ! ./check_date.sh "$(date +%Y-%m-%d)"                 \
+            || ! ./check_date.sh "$(date -d "1 day ago" +%Y-%m-%d)"  \
+            || ! ./check_date.sh "$(date -d "1 day" +%Y-%m-%d)"      \
+            ||   ./check_date.sh "$(date -d "2 days ago" +%Y-%m-%d)" \
+            ||   ./check_date.sh "$(date -d "2 days" +%Y-%m-%d)"
+          then
+            exit 1
+          fi
```

Doesn't GitHub run scripts with `set -e` anyway? Then we could just do this, and get better info about which check failed:
```suggestion
          ! ./check_date.sh "$(date +%Y-%m-%d)"
          ! ./check_date.sh "$(date -d "1 day ago" +%Y-%m-%d)"
          ! ./check_date.sh "$(date -d "1 day" +%Y-%m-%d)"
            ./check_date.sh "$(date -d "2 days ago" +%Y-%m-%d)"
            ./check_date.sh "$(date -d "2 days" +%Y-%m-%d)"
```